src/backport.ts

```diff
@@ -155,20 +155,24 @@ const getFailedBackportCommentBody = ({
   commitSha,
   errorMessage,
   head,
+  repo,
 }: {
   base: string;
   commitSha: string;
   errorMessage: string;
   head: string;
+  repo: string;
 }) => {
-  const worktreePath = `../.worktrees/backport-${base}`;
+  const worktreePath = `../.worktrees/${repo}/backport-${base}`;
   return [
     `The backport to \`${base}\` failed:`,
     "```",
     errorMessage,
     "```",
     "To backport manually, run these commands in your terminal:",
     "```bash",
+    "# Navigate to the root of your repository",
+    "cd $(git rev-parse --show-toplevel)",
```

Comment:
below you switched to `pushd` & `popd`, but here you use `cd` again.
but i think you can just leave the `cd` out - because `git fetch` and `git worktree` work regardless of where you call it as long as you're in the repo. you just need to use an absolute path for the worktree (see comment above)

Reply:
Yeah, this was somewhat intentional because I didn't really want to bury that `git rev-parse --show-toplevel` command as a nested command because I assumed most folks wouldn't be familiar with it (you and I both were not). Here with the comment above it is pretty clear what it is doing.